documentation/src/docs/streamCuts.md

```diff
@@ -0,0 +1,87 @@
+<!--
+Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Working with Pravega: StreamCuts
+
+This section describes about StreamCuts and how it can be used with streaming clients and batch clients.
+Pre-requisites: You should be familiar with Pravega Concepts (see Pravega Concepts).
+
+## Definition
+
+StreamCut represents a consistent position in the stream. It essentially contains a set of segment
+and offset pairs for a single stream which represents the complete keyspace. The offset always points to
+the event boundary and hence there will be no offset pointing to an incomplete event.
+
+Given that data can be continuously added to a stream, the streamcut representing the tail of the
```

Comment:
Here `StreamCut` is referred as "streamcut". Please, make these mentions more consistent.

Reply:
updated all the instances.